src/dbnode/namespace/schema.go

```diff
@@ -278,20 +297,43 @@ func marshalFileDescriptors(fdList []*desc.FileDescriptor) ([][]byte, error) {
 	return dlist, nil
 }
 
-func LoadSchemaRegistryFromFile(schemaReg SchemaRegistry, nsID ident.ID, protoFile string, msgName string, importPathPrefix ...string) error {
-	out, _ := parseProto(protoFile, importPathPrefix...)
+func AppendSchemaOptions(schemaOpt *nsproto.SchemaOptions, protoFile, msgName string, contents map[string]string, deployID string, importPath ...string) (*nsproto.SchemaOptions, error) {
+	sh, err := LoadSchemaHistory(schemaOpt)
```

Comment:
My brain keeps completing this to `shell`, maybe `schemaHist`?

Reply:
ok, it took a while for me realize what you mean.